### ISSUE TITLE
The NEO-D9C module is not supported in v3.2-Mar 8 version 2023.

### ISSUE THREAD

**Issue by pepefrog1234** [sparkfun/SparkFun_RTK_Firmware#449]
### Subject of the issue
I tried to use the UART 2 port for the NEO-D9C to input differential data, but the firmware changes the UART 2 input data type to RTCM 3 at startup, leading to a failure to use differential positioning (CLAS).

### Your workbench
![photo_2023-04-29_09-30-18](https://user-images.githubusercontent.com/32506473/235273097-eab2844b-a4b6-4ffa-96b5-a570d5bfc9c3.jpg)
As shown above, the satellite signal first passes through NEO-D9C to acquire the L6 signal from QZSS and then transmits the signal to NEO-F9P through the feeder.

### Steps to reproduce
The firmware will overwrite the UART 2 settings when it is activated.

### Expected behavior
I would like to add a setting option to adjust the input format of UART 2.
Because NEO-D9C only supports output in UBX format.

### Actual behavior
The firmware will overwrite the UART 2 settings when it is activated.

**Comment by nseidle** [comment]
We don't yet have a product that incorporates the NED-D9C. If you'd like to contribute a PR to support it, we'd be willing to consider it.

**Comment by PaulZC** [comment]
I've added the protocol support in this commit: https://github.com/sparkfun/SparkFun_RTK_Firmware/commit/3bfbd0cb8d9d54d678a210d712eccc44bb57544b

TLDR: This _would_ also allow correction data from an external NEO-D9S to be input over UART2 too. But you would also need to set the encryption keys manually, or automatically, so the ZED could decode the data. And, right now, the key entry and update requires ```online.lband``` to be true.  We would need to significantly restructure the code to support manual key entry without having the NEO-D9S present on the I2C bus. I'm proposing we don't consider that as an option / future upgrade, and close this issue once the protocol support has been merged.

**Comment by PaulZC** [comment]
Closing this. Protocol support was added some time ago...
